Renderer.js

```diff
@@ -0,0 +1,14 @@
+const Renderer = function (data) {
```

Comment:
You shouldn't receive the data once over here, you should send it over every time you invoke the `render` function below.